_episodes/19-access-data.md

```diff
@@ -384,44 +389,41 @@ visual_clip.rio.to_raster("amsterdam_tci.tif", driver="COG")
 > > {: .language-python}
 > >
 > > ~~~
-> > <Item id=LC08_L1TP_013032_20210208_20210305_01_T1>
+> > <Item id=HLS.L30.T18TWL.2021039T153324.v2.0>
 > > ~~~
 > > {: .output}
 > >
 > > ~~~
-> > band_pan_href = item.assets["B8"].href
-> > band_pan = rioxarray.open_rasterio(band_pan_href)
-> > print(band_pan)
+> > print(item.assets["browse"].href)
 > > ~~~
 > > {: .language-python}
 > >
 > > ~~~
-> > <xarray.DataArray (band: 1, y: 15561, x: 15321)>
-> > [238410081 values with dtype=uint16]
-> > Coordinates:
-> >   * band         (band) int64 1
-> >   * x            (x) float64 5.271e+05 5.271e+05 ... 7.569e+05 7.569e+05
-> >   * y            (y) float64 4.582e+06 4.582e+06 ... 4.349e+06 4.349e+06
-> >     spatial_ref  int64 0
-> > Attributes:
-> >     scale_factor:  1.0
-> >     add_offset:    0.0
+> > 'https://data.lpdaac.earthdatacloud.nasa.gov/lp-prod-public/HLSL30.020/HLS.L30.T18TWL.2021039T153324.v2.0/HLS.L30.T18TWL.2021039T153324.v2.0.jpg'
 > > ~~~
 > > {: .output}
 > >
-> > ~~~
-> > bbox = (577000.0, 4504000.0, 597000.0, 4524000.0)
-> > band_pan_clip = band_pan.rio.clip_box(*bbox)
-> > band_pan_clip.squeeze().plot.imshow(figsize=(10,10))
-> > ~~~
-> > {: .language-python}
-> >
-> > <img src="../fig/19-STAC-landsat8-B8-cutout.png" title="Plot of the panchromatic band cutout" alt="landsat8 panchromatic band cutout" width="612" style="display: block; margin: auto;" />
-> >
-> > ~~~
-> > band_pan_clip.rio.to_raster("manhattan.tif", driver="COG")
-> > ~~~
-> > {: .language-python}
+> > ![](../fig/E05-04-STAC-l8-preview.jpg)
 > {: .solution}
 {: .challenge}
 
+> ## Public catalogs, protected data
```

Comment:
Thanks for these good setup notes. I'm thinking we should also include these ahead of any future episode using the CMR stac's assets that require the instructions, and that the instructor during a workshop may walk the students through this setup. Do you have details on why the os env variable is required here? I don't think I've needed to set that before but I'll test it out later.

Reply:
Hi @rbavery, yep, we could then link back to these instructions from future episodes using NASA CMR STAC or copying instructions there. 

Concerning the GDAL options - I am not sure about the detail, as far as I understand these are used to enable the usage of cookie files that gdal/curl need for authentication. On my system, I was not able to fetch data without setting those (either via env variables or via rasterio.Env with something like `with rasterio.Env(GDAL_HTTP_COOKIEFILE="./cookies.txt", ...)`), I am curious to know whether it works on your side. I've noticed also  @srmsoumya sets them in [here](https://github.com/carpentries-incubator/geospatial-python/pull/102).

Reply:
got it, I also had to do all three to be able to access the actual data and not just the metadata.